package.json

```diff
@@ -153,6 +155,7 @@
     "eslint-config-xo-typescript": "^0.19.0",
     "eslint-plugin-header": "^3.0.0",
     "eslint-plugin-prettier": "^3.1.0",
+    "handlebars-helpers": "^0.10.0",
```

Comment:
This should be a dependency, not devDep

Reply:
Making it a dependency makes snyk very unhappy. Let me look a little bit into it.